sycl/CMakeLists.txt

```diff
@@ -217,6 +216,7 @@ install(FILES
 include(AddBoostMp11Headers)
 include(FetchBoostUnorderedHeaders)
 
+set(sycl_inc_dir ${CMAKE_CURRENT_SOURCE_DIR}/include)
```

Comment:
```suggestion
set(sycl_inc_dir ${SYCL_ROOT_SOURCE_DIR}/include)
```

Reply:
Applied in 7123b46c9ba4aad9ad174a2959b181e8c9f765d1